.github/workflows/coverage.yml

```diff
@@ -0,0 +1,68 @@
+name: Code Coverage
+
+on:
+  push:
+    # branches: [ main, develop ]
+  pull_request:
+    # branches: [ main, develop ]
+
+jobs:
+  coverage:
+    runs-on: macos-latest
```

Comment:
I can try again debug and see if I can make it work as expected on linux. For now it's OSX because it worked for me locally with this configuration.